cirq/testing/consistent_decomposition.py

```diff
@@ -22,7 +22,7 @@
 
 def assert_decompose_is_consistent_with_unitary(
         val: Any,
-        including_global_phase: bool=False):
+        including_global_phase: bool=True):
```

Comment:
While we're switching the default value, let's rename this to `ignoring_global_phase` (inverting its meaning).

Reply:
Renamed, hence inverting meaning. So didn't change the default value.